app/models/pregnancy.rb

```diff
@@ -7,9 +7,9 @@ class Case
 
 	field :line, type: String #DC, MD, VA
 	field :language, type: String
-	field :case_id, type: String
+	field :p_case_id, type: String
```

Comment:
@Kevin-Wei I think we can switch these up to `pregnancy_id` and `pregnancy_status`, right? Or would it be better to prepend the `p` here?